RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -685,9 +712,10 @@ void MuonIdProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
      reco::MuonTime muonTime;
      reco::MuonTimeExtra dtTime;
      reco::MuonTimeExtra cscTime;
+     reco::MuonTime rpcTime;
      reco::MuonTimeExtra combinedTime;
 
-     theTimingFiller_->fillTiming(muon, dtTime, cscTime, combinedTime, iEvent, iSetup);
+     theTimingFiller_->fillTiming(muon, dtTime, cscTime, rpcTime, combinedTime, iEvent, iSetup);
```

Comment:
looking at your commit history there is a back-and-forth with the old and new version of these lines.
a rebase (to 800 or at least pre6) may help (if you manage to get past the merge commit in the middle of your branch .. there is just one of them, which likely means the rebase will work).